_layouts/report.md

```diff
@@ -148,4 +148,4 @@
             <br><br>
         </div>
     </body>
-</html> 
+</html>
```

Comment:
Please don't do unnecessary changes, it's not the scope of this PR.
You can fix this later.